using-mobile-extensions/adobe-analytics/README.md

```diff
@@ -122,21 +122,24 @@ You may do the following after calling the `setApplication()` method in the `onC
 ```java
 public class MobileApp extends Application {
 
- @Override
- public void onCreate() {
-     super.onCreate();
-     MobileCore.setApplication(this);
-     MobileCore.ConfigureWithAppId("yourAppId");
-     try {
-         Analytics.registerExtension(); //Register Analytics with Mobile Core
-         Identity.registerExtension();
-         MobileCore.start(null);
-     } catch (Exception e) {
-         //Log the exception
-     }
- }
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        MobileCore.setApplication(this);
+        MobileCore.ConfigureWithAppId("yourAppId");
+        try {
+            Analytics.registerExtension(); //Register Analytics with Mobile Core
+            Identity.registerExtension();
+            MobileCore.start(null);
+        } catch (Exception e) {
+            //Log the exception
+         }
+    }
 }
 ```
+
+**Important**: Analytics depends on the Identity extension and is automatically included in Core via Maven. When installing manually, ensure that you have also added the `identity-1.x.x.aar` to your project.
```

Comment:
- Please reword the 2nd sentence as follows:

"When installing the Analytics extension manually, ensure that you added the `identity-1.x.x.aar` to your project."